packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -106,7 +106,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
     FinderInterface private immutable finder;
 
     // Reference to Slashing Library, used to compute slashing amounts.
-    SlashingLibraryInterface public slashingLib;
+    SlashingLibraryInterface public slashingLibrary;
```

Comment:
this is a cosmetic revert. better to call it `slashingLibrary` over `slashingLib`